lookout/style/format/langs/__main__.py

```diff
@@ -28,6 +36,12 @@ def parse_args():
     #
     # Use find | xargs instead.
     parser.add_argument("input", nargs="+", help="Paths to the sample files.")
+    parser.add_argument("--parquet", action="store_true", default=False,
+                        help="Set the input format to parquet, Must have columns: "
+                             "'path', 'content', 'uast'")
+    languages = ["java", "python", "go", "javascript", "typescript", "ruby", "bash", "php"]
+    parser.add_argument("-l", "--language", choices=languages, default="",
+                        help="The programming languages to analyse.")
```

Comment:
Sorry, missed this:

`The programming language to analyse.` - single language always, no "s"

Reply:
Oh yes now I get it why I asked that! Indeed, we need `--parquet-language` because we always specify the paths and the user can filter them for each language in every way possible - and they should. We don't do any recursive scanning inside.

+ change the help message accordingly